doc/user_guide/administration.md

```diff
@@ -0,0 +1,375 @@
+## Administration
```

Comment:
Are this the administrative steps, the DB admin, or the schema administrator, or both need to take care of?
e.g. DB admin + activating RLS feature, schema admin installing the scripts for his schema etc.?

which "role" is required for the task(s): DB admin, schema admin

Reply:
Added explantion.